plugins/fianu/README.md

```diff
@@ -0,0 +1,42 @@
+# [Fianu](https://www.fianu.io/)
```

Comment:
Fianu needs to be added to the `.github/vale/Vocab/Backstage/accept.txt`

Reply:
done